### PR TITLE
Add ICDS citusdb vms

### PR DESCRIPTION
**Pull request by sravfeyn**
##### SUMMARY

Adding the new 26 TCL Citus VMs to the inventory

##### ENVIRONMENTS AFFECTED
icds

##### ISSUE TYPE
- Change Pull Request

##### COMPONENT NAME
ICDS Inventory

### CODE REVIEW

**Comment by sravfeyn** [comment]
@snopoke

File: environments/icds-cas/inventory/citusdb.csv [hunk 1]
```diff
@@ -0,0 +1,27 @@
+hostname,host_address,group 1,group 2,group 3,var: hostname
+Cituscoordinator0,100.71.184.115,citusdb,citusdb_cordinator,,MUMGCCWCDPRDCC00
```
Comment (snopoke) [inline environments/icds-cas/inventory/citusdb.csv]: can we make these lowercase. How about `citusmaster`? Its shorter and easier to type...

Comment (sravfeyn) [reply environments/icds-cas/inventory/citusdb.csv]: 👍

**Comment by sravfeyn** [comment]
Re updating the postgres config, I didn't update since we likely would revisit the config, but I could base the config based off of `ucr_dbs` config and the fact that all incoming connections connect to coordinator node and not directly to worker node.

**Comment by sravfeyn** [comment]
Updated @snopoke

**Comment by snopoke** [comment]
@sravfeyn how did you deploy citus without this: https://github.com/dimagi/commcare-cloud/pull/2913

**Comment by sravfeyn** [comment]
(offlined) Deploying again.

**Pull request merged**